docs/gradle.md

```diff
@@ -19,19 +19,21 @@ For Android projects, the plugin will create a default database called `Database
 `build.gradle`:
 ```groovy
 sqldelight {
+  // Database name
   MyDatabase {
-    //package name used for the generated MyDatabase.kt
+    // Package name used for the generated MyDatabase.kt
     packageName = "com.example.db"
 
-    // An array of folders where the plugin will read your '.sq' and '.sqm' files.
-    // The folders are relative to the existing source set so if you specify ["db"],
-    // the plugin will look into 'src/main/db'
+    // An array of folders where the plugin will read your '.sq' and '.sqm' 
+    // files. The folders are relative to the existing source set so if you
+    // specify ["db"], the plugin will look into 'src/main/db'. 
     // Defaults to ["sqldelight"] (src/main/sqldelight)
     sourceFolders = ["db"]
 
-    // The directory where to store '.db' schema files relative to the root of the project.
-    // These files are used to verify that migrations yield a database with the latest schema.
-    // Defaults to null so the verification tasks will not be created.
+    // The directory where to store '.db' schema files relative to the root 
+    // of the project. These files are used to verify that migrations yield 
+    // a database with the latest schema. Defaults to null so the verification 
+    // tasks will not be created.
```

Comment:
The previous lines were too long to fit in the available space and required scrolling. Here's a screenshot of how this code block will look with these changes:

![Screen Shot 2019-12-13 at 2 28 03 PM](https://user-images.githubusercontent.com/2387680/71120938-607ab480-21ab-11ea-9874-fd3b72fef925.png)